modules/standard/Reflection.chpl

```diff
@@ -220,46 +331,96 @@ proc getFieldRef(ref x:?t, param s:string) ref {
   return __primitive("field by num", x, i);
 }
 
+/* Get a field index in a class or record, or -1 if
+   the field is not found.
+
+   :arg t: a class or record type
+   :arg name: the name of a field
+   :returns: an index `i` usable in getField, or -1 if the field was not found.
+ */
+proc getFieldIndex(type t, param name:string) param : int
+  return __primitive("field name to num", checkQueryT(t), name)-1;
+
 /* Get a field index in a class or record, or -1 if
    the field is not found.
 
    :arg t: a class or record type
    :arg s: the name of a field
    :returns: an index `i` usable in getField, or -1 if the field was not found.
  */
+pragma "last resort"
+deprecated "The formal 's' is deprecated, please use 'name' instead"
 proc getFieldIndex(type t, param s:string) param : int
   return __primitive("field name to num", checkQueryT(t), s)-1;
 
+/* Returns `true` if a class or record has a field named `name`,
+   or `false` otherwise.
```

Comment:
true and false should have two backtics on either side instead of just one - the current set up will italicize them in the documentation instead of turning them into a code block (it's really easy to forget cuz it's different than Github syntax)